tests/flow/object_api/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -182,14 +182,18 @@ var sealed = { one: \"one\", two: \"two\" };
 
 // error, Array<string>
 var unsealed = {};
-Object.keys(unsealed).forEach(k => {
-  (k: number); // error: number ~> string
-});
+Object
+  .keys(unsealed)
+  .forEach(k => {
+    (k: number); // error: number ~> string
+  });
```

Comment:
This is very common pattern that we really shouldn't break on, as mentioned above. In fact `Object.keys` was the main reason I tried to avoid breaking on it because this looks pretty ugly compared to before.